frontend/src/components/LoginCard.jsx

```diff
@@ -19,6 +19,12 @@ import { useSetRecoilState } from "recoil";
 import authScreenAtom from "../atoms/authAtom";
 import useShowToast from "../hooks/useShowToast";
 import userAtom from "../atoms/userAtom";
+import { z } from "zod";
+
+const loginSchema = z.object({
+    username: z.string().min(1, "Username is required."),
+    password: z.string().min(1, "Password is required."),
```

Comment:
Check backend model to see field requirement
![image](https://github.com/QuocZuong/threads/assets/110612155/489ab9a7-94b0-48ff-b014-a65ebd21ca08)